lib/commands/find.js

```
@@ -0,0 +1,31 @@
+import { util } from 'appium-support';
+
+
+const commands = {};
+
+commands.findElOrEls = async function findElOrEls (strategy, selector, mult, context) {
+  context = util.unwrapElement(context);
+  const endpoint = `/element${context ? `/${context}/element` : ''}${mult ? 's' : ''}`;
+
+  // Should we do this workaround or just let it fail?
```

i'm fine to just let it fail, we should encourage people to be moving into the future and this is a new project

---

i don't mind. I was just thinking about Appium Desktop, where we still show ids. Would that be fine?